### PR TITLE
Feature/fzf

### PR DESCRIPTION
**Pull request by timhourigan**
## What's changed

- Add fzf
- Add Makefile structure
- Add Molecule test infrastructure

## Checklist

- [x] README.md updated
- [x] CHANGELOG.md updated

### CODE REVIEW

File: roles/fzf/meta/main.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+galaxy_info:
+  author: Tim Hourigan
+  description: Install fzf
+  license: MIT
+  min_ansible_version: 2.5
+  role_name: fzf
+  namespace: timhourigan
+  platforms:
```
Comment (darrenmeehan) [inline roles/fzf/meta/main.yml:8]: From a first pass of the code it looks like this also supports Fedora. Is it worth adding it here?

File: roles/fzf/handlers/main.yml [hunk 2]
```diff
@@ -0,0 +1 @@
+---
```
Comment (darrenmeehan) [inline roles/fzf/handlers/main.yml:1]: I've tended to not add files or directories for Ansible features I'm not using.

I also tend to forget they exist though, so having them included is a good idea.

Just some thoughts, I've yet to figure out a convention for these myself!

**Comment by darrenmeehan** [comment]
Thanks for sharing this code change, it's quite the setup! Molecule is something I must try and implement myself.

Much better than testing out my playbooks on home infrastructure I'd prefer not to break!

File: roles/fzf/handlers/main.yml (hunk 2 above)
Comment (timhourigan) [reply roles/fzf/handlers/main.yml:1]: I'm 50/50 on this. I even had them removed, in an earlier commit.

But as I'll probably use this role as a template for others, I ended up leaving them in. I might end up removing them again, as I add other roles.

File: roles/fzf/meta/main.yml (hunk 1 above)
Comment (timhourigan) [reply roles/fzf/meta/main.yml:8]: See below :smile: 

I'm going to try to support Debian, Fedora (34 for now) and Ubuntu.

Comment (timhourigan) [reply roles/fzf/meta/main.yml:8]: You can see from the molecule file, that all 3 flavours are being tested.

**Pull request merged**